go/kbcmf/decrypt.go

```diff
@@ -27,6 +27,9 @@ type decryptStream struct {
 	keys       *receiverKeysPlaintext
 	sessionKey SymmetricKey
 	buf        []byte
+	nonce      *Nonce
+	macKey     BoxPrecomputedSharedKey
```

Comment:
Maybe call this "diffieHellmanSharedSecret" or "boxBeforenm" or something? It took me a second to figure out what it was.